engines/ags/lib/allegro/gfx.cpp

```diff
@@ -147,7 +147,11 @@ BITMAP *create_bitmap_ex(int color_depth, int width, int height) {
 		format = Graphics::PixelFormat(2, 5, 6, 5, 0, 11, 5, 0, 0);
 		break;
 	case 32:
+#ifdef USE_OPENGL
+		format = Graphics::PixelFormat(4, 8, 8, 8, 8, 0, 8, 16, 24);
+#else
```

Comment:
Does this change really make sense? We could either keep bitmaps in their original format and do the pixel conversion when blitting to the game surface, or we could do the conversion when loading the data, but this does not seem to be included in your changes.

Reply:
I am working on a replacement FakeTexture anyways, that allows to render with the right colors. That makes this osolete. I just inserted it, because otherwise AGS on Android would not run at all because it would not detect the right format.

Reply:
Will open new PR with PixelFormat fixes.